server/build/babel/preset.js

```diff
@@ -30,14 +40,14 @@ module.exports = {
       require.resolve('babel-plugin-module-resolver'),
       {
         alias: {
-          'babel-runtime': babelRuntimePath,
-          'next/link': require.resolve('../../../lib/link'),
-          'next/prefetch': require.resolve('../../../lib/prefetch'),
-          'next/css': require.resolve('../../../lib/css'),
-          'next/head': require.resolve('../../../lib/head'),
-          'next/document': require.resolve('../../../server/document'),
-          'next/router': require.resolve('../../../lib/router'),
-          'next/error': require.resolve('../../../lib/error')
+          'babel-runtime': packageRelativeResolve('babel-runtime/package'),
+          'next/link': packageRelativeResolve('../../../lib/link'),
+          'next/prefetch': packageRelativeResolve('../../../lib/prefetch'),
+          'next/css': packageRelativeResolve('../../../lib/css'),
+          'next/head': packageRelativeResolve('../../../lib/head'),
+          'next/document': packageRelativeResolve('../../../server/document'),
+          'next/router': packageRelativeResolve('../../../lib/router'),
+          'next/error': packageRelativeResolve('../../../lib/error')
```

Comment:
I agree about `next/*`.

So something like the following ?

- `/somewhere/app/node_modules/babel-runtime/...` => `babel-runtime/...`
- `/somewhere/app/node_modules/next/node_modules/babel-runtime/...` => `next/node_modules/babel-runtime/...`

It seems smart 😄